common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -117,6 +117,14 @@ class ProctoringFields(object):
     )
 
     @property
+    def is_timed_exam(self):
+        """
+        Alias the permutation of above fields that corresponds to un-proctored timed exams
+        to the more clearly-named is_timed_exam
+        """
+        return not self.is_proctored_enabled and self.is_time_limited
```

Comment:
I believe this needs to be `return not self.is_proctored_enabled and not self.is_practice_exam and self.is_time_limited`